test/functional/hashdump_v2/hashdump-file-hash.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+global_setup() {
+
+	create_test_environment "$TEST_NAME"
+	printf "test-data" | sudo tee "$TEST_NAME"/target-dir/test-hash > /dev/null
+
+}
+
+test_setup() {
+
+	return
+
+}
+
+test_teardown() {
+
+	return
+
+}
+
+global_teardown() {
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+@test "hashdump with prefix" {
+
+	run sudo sh -c "$SWUPD hashdump --path=$TEST_NAME/target-dir /test-hash"
```

Comment:
Extra space in "--path=$TEST_NAME/target-dir /test-hash"" ?
-----------------------------------------------------------^

Reply:
you mean between "--path=$TEST_NAME/target-dir" and "/test-hash", that is the way it is supposed to be, $TEST_NAME/target-dir are part of the --path argument, /test-hash is a different argument